pkgs/development/python-modules/cgroup-utils/default.nix

```diff
@@ -29,11 +18,27 @@ buildPythonPackage rec {
     sha256 = "0qnbn8cnq8m14s8s1hcv25xjd55dyb6yy54l5vc7sby5xzzp11fq";
   };
 
-  meta = with lib; {
+  postPatch = ''
+    sed -i -e "/argparse/d" setup.py
+  '';
+
+  build-system = [ setuptools ];
+
+  buildInputs = [ pep8 ];
```

Comment:
This is just a `checkInput` for code style, and we don’t run the tests, so it can probably be dropped.